Makefile

```diff
@@ -39,9 +39,12 @@ setup_lint:
 	python tools/actions_local_runner.py --file .github/workflows/lint.yml \
 	 	--job 'mypy' --step 'Install dependencies' --no-quiet
 
-# TODO: This is broken on MacOS (it downloads a Linux binary)
-	python tools/actions_local_runner.py --file .github/workflows/lint.yml \
-	 	--job 'quick-checks' --step 'Install ShellCheck' --no-quiet
+	@if [ "$$(uname)" = "Darwin" ]; then \
+		brew install shellcheck; \
```

Comment:
this requires a couple additional steps on FB machines ime

Reply:
I usually have to `sudo chown -R (whoami) /usr/local/bin /usr/local/lib /usr/local/sbin`, but with the message about installing `brew` + brew telling you to `chown` it should be pretty clear how to fix the errors